### PR TITLE
Credits89

### PR DESCRIPTION
**Pull request by mattam82**
This is a first draft of credits for 8.9. Given the large number of user-visible changes (many deprecations in particular), I'm torn between listing all of them in the credits section (basically copying CHANGES.md) or being more selective. What do you guys think? It would be ok to merge this switfly so that it is part of 8.9beta and refine it until the release, if I did not make an enormous mistake somewhere :)

### CODE REVIEW

File: doc/sphinx/credits.rst [hunk 1]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+--------------------
+
+|Coq| version 8.9 contains the result of refinements and stabilization
+of features and deprecations or removals of deprecated features,
+cleanups of the internals of the system and API along with a few new
+features. This release includes many user-visible changes, including
+deprecations that are documented in ``CHANGES.md``.
+
+TODO:
+We only list the most important ones:
```
Comment (silene) [inline doc/sphinx/credits.rst]: Suggestion: Here are the most important ones.

File: doc/sphinx/credits.rst [hunk 2]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+or
+The main user visible changes are:
+
+- Kernel: mutually defined records are now supported, by P.M. Pédrot.
+
+- Notations: support for autonomous grammars of terms called "custom
+  entries", by H. Herbelin (see chapter "Syntax extensions" of the
+  reference manual).
+
+- Notations: deprecated notations will be removed in the next version of
```
Comment (silene) [inline doc/sphinx/credits.rst]: "deprecated notations of the standard library will be"

File: doc/sphinx/credits.rst [hunk 3]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+  |Coq|, see the``CHANGES.md`` file for a script to ease porting,
+  by J. Gross.
+
+- Tactics: Introduction tactics `intro`/`intros` on a goal that is an
```
Comment (silene) [inline doc/sphinx/credits.rst]: I am not fluent with reStructuredText, but shouldn't those be double backquotes? (And similarly below.)

File: doc/sphinx/credits.rst [hunk 4]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+- Vernacular:
+
+  - Experimental support for attributes on commands by V. Laporte, as in
+    `#[local] Lemma foo : bar.`
+    Tactics and tactic notations now support the `deprecated` attribute.
+
+  - Removed deprecated commands `Arguments Scope` and `Implicit
+    Arguments` in favor of `Arguments`.
+
+  - New option `Uniform Inductive Parameters` by J. Hugunin to avoid
```
Comment (silene) [inline doc/sphinx/credits.rst]: "New command Set Uniform..." by homogeneity with the following one?

File: doc/sphinx/credits.rst [hunk 5]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+cleanups of the internals of the system and API along with a few new
+features. This release includes many user-visible changes, including
+deprecations that are documented in ``CHANGES.md``.
+
+TODO:
+We only list the most important ones:
+or
+The main user visible changes are:
+
+- Kernel: mutually defined records are now supported, by P.M. Pédrot.
```
Comment (silene) [inline doc/sphinx/credits.rst]: You switch to full first name later on.

File: doc/sphinx/credits.rst [hunk 6]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+    Arguments` in favor of `Arguments`.
+
+  - New option `Uniform Inductive Parameters` by J. Hugunin to avoid
+    repeating uniform parameters in constructor declarations.
+
+  - New commands `Set Hint Variables/Constants Opaque/Transparent`
+    commands for controlling the opacity flag of variables and constants
+    in hint databases.
+
+  - Added the `Numeral Notation` command for registering decimal numeral
```
Comment (JasonGross) [inline doc/sphinx/credits.rst]: Perhaps this should be classed with notations?

File: doc/sphinx/credits.rst (hunk 1 above)
Comment (mattam82) [reply doc/sphinx/credits.rst]: Ok

File: doc/sphinx/credits.rst (hunk 3 above)
Comment (mattam82) [reply doc/sphinx/credits.rst]: In CHANGES.md they're simple. @Zimmi48 ?

File: doc/sphinx/credits.rst (hunk 6 above)
Comment (mattam82) [reply doc/sphinx/credits.rst]: Indeed, I made a grouping of all Notations-related changes

File: doc/sphinx/credits.rst (hunk 2 above)
Comment (mattam82) [reply doc/sphinx/credits.rst]: Right

**Comment by mattam82** [comment]
Gah I pushed to the main branch by mistake. Feel free to revert

**Comment by Zimmi48** [comment]
Why was this targeting `v8.9` anyways?

**Review by Zimmi48 — COMMENTED** [review]
I'm happy with this way of formulating the credits but this reinforces my belief that we should consolidate all the release notes into a single document. See also #7058 and #7075 (and https://about.gitlab.com/2018/07/03/solving-gitlabs-changelog-conflict-crisis/ which is on my reading list).

File: doc/sphinx/credits.rst [hunk 7]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+
+TODO:
+We only list the most important ones:
+or
+The main user visible changes are:
+
+- Kernel: mutually defined records are now supported, by P.M. Pédrot.
+
+- Notations: support for autonomous grammars of terms called "custom
+  entries", by H. Herbelin (see chapter "Syntax extensions" of the
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/credits.rst]: Right, I referenced where I could find links in the rest.

File: doc/sphinx/credits.rst [hunk 8]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+  reference manual).
+
+- Notations: deprecated notations will be removed in the next version of
+  |Coq|, see the``CHANGES.md`` file for a script to ease porting,
+  by J. Gross.
+
+- Tactics: Introduction tactics `intro`/`intros` on a goal that is an
+  existential variable now force a refinement of the goal into a
+  dependent product rather than failing.
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/credits.rst]: I added them where it was missing.

File: doc/sphinx/credits.rst [hunk 9]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+
+- Decision procedures: deprecation of tactics `fourier` and `romega` in
+  favor of `lra` and `lia` respectively.
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/credits.rst]: Ok

File: doc/sphinx/credits.rst (hunk 3 above)
Comment (mattam82) [reply doc/sphinx/credits.rst]: You were right, for literal code `` `` is the right syntax

File: doc/sphinx/credits.rst [hunk 10]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+- Toplevels: `coqtop` and `coqide` can now display diffs between proof
+  steps in color, using the `Set Diffs` option, by J. Fehrle.
+
+- Documentation: we integrated a large number of fixes to the new Sphinx
+  documentation.
+
+- Tools: removed the `gallina` utility and the homebrewed `Emacs` mode.
+
+- Packaging: Michael Soegtrop maintains a compilation chain for Windows
+  allowing to build Coq with optional packages, including MTac2, LTac2 and
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/credits.rst]: Right

File: doc/sphinx/credits.rst [hunk 11]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+
+- Tactics: Introduction tactics `intro`/`intros` on a goal that is an
+  existential variable now force a refinement of the goal into a
+  dependent product rather than failing.
+
+- Decision procedures: deprecation of tactics `fourier` and `romega` in
+  favor of `lra` and `lia` respectively.
+
+- Proof language: focusing bracket `{` now supports named goal selectors,
+  e.g. `[x]: {` will focus on a goal (existential variable) named `x`.
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/credits.rst]: Yep, I did that

**Comment by mattam82** [comment]
I was misguided in targeting 8.9 indeed. I addressed all comments and checked the output is nice, with links as far as I could find places to link to.

**Comment by mattam82** [comment]
Superseded by PR# 8851 I think we should simply revert my changes to the 8.9 branch.

File: doc/sphinx/credits.rst [hunk 12]
```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
...
+
+- Ltac: Support for `fix`/`cofix` added in Ltac `match` and `lazymatch`.
+
+- SSReflect: delayed clear switches have a simpler semantics and empty
+  clear switches are allowed before view applications of variable views.
+
+- Vernacular:
+
+  - Experimental support for attributes on commands by V. Laporte, as in
```
[the remark this answers is not in the document]
Comment (ejgallego) [reply doc/sphinx/credits.rst]: Maybe it is better just to omit it then, unless we think we really want user feedback.

**Pull request merged**